pkg/plugins/plugins.go

```diff
@@ -387,6 +499,18 @@ func (p *Plugin) IsExternalPlugin() bool {
 	return p.Class == External
 }
 
+func (p *Plugin) Manifest() []byte {
+	// nolint:gosec
+	// We can ignore the gosec G304 warning on this one because `manifestPath` is based
+	// on plugin the folder structure on disk and not user input.
+	d, err := os.ReadFile(filepath.Join(p.PluginDir, "MANIFEST.txt"))
```

Comment:
Possible to get rid of the nolint using what we've used for plugin dashboards?
https://github.com/grafana/grafana/blob/13d5ad2ce2a7283dc25dace1328cbf0c11fec639/pkg/plugins/manager/dashboards/filestore.go#L87-L94